queries/flights.py

```diff
@@ -385,8 +385,10 @@ def _format_flight_answer(flights: FlightList) -> Dict[str, str]:
 
     for flight in flights:
         assert isinstance(flight.get("No"), str)
+        assert isinstance(flight.get("DisplayName"), str)
```

Comment:
These should not be needed, with judicious use of .get() instead of []

Reply:
okidoki, fixed!